governance/program/src/state/governance.rs

```diff
@@ -205,21 +209,30 @@ impl GovernanceV2 {
         governing_token_mint: &Pubkey,
     ) -> Result<(), ProgramError> {
         // resolve_vote_threshold() asserts the vote threshold exists for the given governing_token_mint and is not disabled
-        let _ = self.resolve_vote_threshold(realm_data, governing_token_mint)?;
+        let _ = self.resolve_vote_threshold(realm_data, governing_token_mint, None)?;
 
         Ok(())
     }
 
-    /// Resolves VoteThreshold for the given realm and governing token
+    /// Resolves VoteThreshold for the given realm, governing token and optional Vote
     pub fn resolve_vote_threshold(
         &self,
         realm_data: &RealmV2,
         governing_token_mint: &Pubkey,
+        vote: Option<&Vote>,
```

Comment:
Please split this one up as well or make the parameter mandatory, as noted above.

Reply:
Fixed with `VoteKind`